exercise/templates/exercise/_exercise_info.html

```diff
@@ -38,6 +38,10 @@ <h4 class="panel-title">{% trans "Exercise info" %}</h4>
             </dd>
             {% endif %}
 
+            <dt>{% trans "Exercise category" %}</dt>
```

Comment:
Categories are not always visible for the students. Therefore, this should be checked before rendering the name of the category.